books/RayTracingInOneWeekend.html

```diff
@@ -1446,12 +1457,195 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
             return 0.5 * (rec.normal + color(1,1,1));
         }
+
+        vec3 unit_direction = unit_vector(r.direction());
+        auto a = 0.5*(unit_direction.y() + 1.0);
+        return (1.0-a)*color(1.0, 1.0, 1.0) + a*color(0.5, 0.7, 1.0);
+    }
     ...
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [main-with-interval]: <kbd>[main.cc]</kbd> The new main using interval]
 
 
 
+A New Camera Class
+====================================================================================================
+Before continuing, now is a good time to consolidate our camera and scene-render code into a single
+  new class: `camera`.
+The camera class will be responsible for constructing and dispatching rays into the world, and using
+  the results to construct the rendered image.
```

Comment:
The camera class will be responsible for two important jobs. First, it will construct and then dispatch rays into the world. Second, it will use the results of dispatched rays to construct the rendered image.

Reply:
Done.